chat_shell/chat_shell/api/v1/response.py

```
@@ -39,16 +36,62 @@
+    to yield them as SSE events in real-time.
     """
 
+    def __init__(self):
+        """Initialize SSE transport with an async queue."""
+        self._queue: asyncio.Queue[tuple[str, dict]] = asyncio.Queue()
+        self._done = False
+
+    async def send(
+        self,
+        event_type: str,
+        task_id: int,
```

Ruff (0.15.0)

[warning] Unused method argument: `task_id`

(ARG002)